cachi2/core/rooted_path.py

```diff
@@ -70,8 +70,9 @@ def subpath_from_root(self) -> Path:
         return self._path.relative_to(self._root)
 
     def __eq__(self, other: object) -> bool:
-        # Note: __eq__ is implemented mainly for unit tests that do assert_called_with() or similar
         if not isinstance(other, RootedPath):
+            # NotImplemented is a special value which should be returned by the binary special methods
```

Comment:
You omitted the important part on which binary methods - **rich comparison methods**.

While at that, you could also tweak the commit subject ever so slightly to say that we're adding ignore patterns rather than increasing the overall coverage in any way as the way it's written now can lead to misunderstandings.

Reply:
Fixed; I copied one more line from the docs for further explanation.